pep-0612.rst

```diff
@@ -0,0 +1,258 @@
+PEP: 612
+Title: Parameter Specification Variables
+Author: Mark Mendoza <mendoza.mark.a@gmail.com>
+Discussions-To: Typing-Sig <typing-sig@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 18-Dec-2019
+Python-Version: 3.9
+Post-History: 18-Dec-2019
+
+Parameter Specification Variables
+=================================
+
+Abstract
+--------
+
+There currently are two ways to specify the type of a callable, the ``Callable[[T1, T2], TReturn]`` syntax defined in  `PEP 484 <https://www.python.org/dev/peps/pep-0484>`_\ , and callback protocols from `PEP 544 <https://www.python.org/dev/peps/pep-0544/#callback-protocols>`_. Neither of these support forwarding the parameter types of one callable over to another callable, making it difficult to annotate function decorators. This PEP proposes ``typing.ParameterSpecification``\ , a new kind of type variable, to support expressing these kinds of relationships. 
```

Comment:
I honestly prefer if the raw text of a PEP is wrapped at something like 70-80 columns (with an exception for long URLs etc.).

PS. Why the backslash-space before commas?

Reply:
re: the escaping, I'm not exactly sure, as they were generated by this markdown -> rst converter (https://pypi.org/project/m2r/), but removing them causes this error
```
 (WARNING/2) Inline literal start-string without end-string.
```

Reply:
I wrapped at 80 in https://github.com/python/peps/pull/1260.